packages/fiori/src/ProductSwitch.js

```diff
@@ -156,6 +156,13 @@ class ProductSwitch extends UI5Element {
 		}
 	}
 
+	handleProductSwitchItemClick(event) {
+		event.target.parentNode.querySelectorAll("ui5-product-switch-item").forEach(item => {
```

Comment:
To get the ProductSwitch items, you just need to use "this.items" and this will become 
this.items.forEach(item => { item.selected = false;});

Reply:
Should not this behaviour also happen on SPACE/ENTER?